src/main/resources/admin/tools/content-type-manager/modules/module-example.html

```diff
@@ -127,11 +127,11 @@
             const desiredData = _.get(state.json, desiredPath);
             const currentData = _.get(state.json, currentPath);
 
-            if (
+            /* if (
```

Comment:
Unused code can be deleted. 
Unless you know that it will be used later on?

Reply:
Thanks, I'll delete it!